client/lib/routing/roter.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:client/routing/routes.dart';
```

Comment:
The file is named "roter", don't you mean "router" instead?

Reply:
my bad! I don't why I do such silly mistakes